src/main/java/org/folio/circulation/services/PubSubPublishingService.java

```diff
@@ -22,16 +24,15 @@
   private static final Logger logger = LoggerFactory.getLogger(PubSubPublishingService.class);
 
   private final Map<String, String> okapiHeaders;
-  private final PubsubClient pubSubClient;
+  private final Context vertxContext;
 
   public PubSubPublishingService(RoutingContext routingContext) {
     this(new WebContext(routingContext));
```

Comment:
Maybe we can retrieve Vertx from `routingContext.vertx()`

Reply:
There's another public constructor: `public PubSubPublishingService(WebContext context) {`